boards/renesas/ek_ra2l1/ek_ra2l1_defconfig

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2021-2024 MUNIC SA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
+
+CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_BUILD_NO_GAP_FILL=y
```

Comment:
UART device is required

Reply:
This is the scope of the following PRs (still under hot discussion). But unfortunately, based on this root PR.